src/Dropdown/Dropdown.js

```diff
@@ -28,6 +28,13 @@ export default class Dropdown extends Util.mixin(BindMixin) {
     };
   }
 
+  componentWillReceiveProps(nextProps) {
+    // Set the selectedID to the forceSelectedID if set
+    if (this.props.forceSelectedID) {
+      this.setState({selectedID: nextProps.forceSelectedID});
+    }
```

Comment:
Why store it in state? If it's already in props can't we just always look there?